nimble/host/store/config/src/ble_store_config.c

```diff
@@ -101,16 +101,6 @@ ble_store_config_find_sec(const struct ble_store_key_sec *key_sec,
             }
         }
 
-        if (key_sec->ediv_rand_present) {
-            if (cur->ediv != key_sec->ediv) {
-                continue;
-            }
-
-            if (cur->rand_num != key_sec->rand_num) {
-                continue;
-            }
-        }
-
         if (key_sec->idx > skipped) {
```

Comment:
this is also not needed since we can have only one set of keys for device (unlike cccd)

also remove all unused members from ble_store_key_sec structure (everything except peer_addr)